### PR TITLE
chore(generator): remove fetch sample tag

### PR DESCRIPTION
**Pull request by yukun-dong**

### CODE REVIEW

File: packages/fx-core/src/component/generator/generator.ts [hunk 1]
```diff
@@ -82,7 +84,7 @@ export class Generator {
       destination: destinationPath,
       logProvider: ctx.logProvider,
       zipUrl: sample.link,
```
Comment (hund030) [inline packages/fx-core/src/component/generator/generator.ts:86]: `sample.link ?? sampleConfig.defaultPackageLink`?

File: packages/fx-core/src/component/generator/utils.ts [hunk 2]
```diff
@@ -135,19 +121,6 @@ export async function fetchTemplateZipUrl(
   return `${templateConfig.templateDownloadBaseURL}/${selectTemplateTag}/${name}.zip`;
```
Comment (hund030) [inline packages/fx-core/src/component/generator/utils.ts:121]: Seems there's a bug. It should be `${templateConfig.templateDownloadBaseURL}/${selectedTag}/${name}.zip`

File: packages/fx-core/src/component/generator/generator.ts (hunk 1 above)
Comment (yukun-dong) [reply packages/fx-core/src/component/generator/generator.ts:86]: This logic is in samples.ts. https://github.com/OfficeDev/TeamsFx/blob/7f7c9560f1f5a13bb57eeb7a61feb29da8998d30/packages/fx-core/src/common/samples.ts#L35

File: packages/fx-core/src/component/generator/utils.ts (hunk 2 above)
Comment (yukun-dong) [reply packages/fx-core/src/component/generator/utils.ts]: Thanks. Fixed.

File: packages/fx-core/src/common/samples-config.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
-    "baseUrl": "https://github.com/OfficeDev/TeamsFx-Samples/tree/v1.0.0/",
-    "defaultPackageLink": "https://github.com/OfficeDev/TeamsFx-Samples/archive/refs/tags/v1.0.0.zip",
+    "baseUrl": "https://github.com/OfficeDev/TeamsFx-Samples/tree/v1.1.0/",
+    "defaultPackageLink": "https://github.com/OfficeDev/TeamsFx-Samples/archive/refs/tags/v1.1.0.zip",
```
Comment (hund030) [inline packages/fx-core/src/common/samples-config.json:3]: Why do we upgrade the sample version?

Comment (yukun-dong) [reply packages/fx-core/src/common/samples-config.json:3]: The sample version is updated by spring release, not by me.

Comment (hund030) [reply packages/fx-core/src/common/samples-config.json:3]: Then you may need to clean up your commits. It's better if your PR only includes your changes.

Comment (yukun-dong) [reply packages/fx-core/src/common/samples-config.json:3]: Thanks. Updated.

**Review by hund030 — APPROVED** [review]

**Pull request merged**